External/SPMmodified/xASL/xASL_im_FillNaNs.m

```diff
@@ -73,18 +70,23 @@
     bQuality = 1; % default
 end
 
-if UseMethod==1 % only method that needs VoxelSize
-    if nargin<4 || isempty(VoxelSize)
+if UseMethod == 1 || UseMethod == 3 % only methods that needs VoxelSize
```

Comment:
That's just a misunderstanding:
- CAT12 can indeed have different voxel-sizes and we don't have the NII.
- So that's why I estimate the voxel-size automatically as we agreed by looking at the deformation field.
- But to make this a bit transparent, I save the estimated voxel-size to the VoxelSize field.
- But in theory, you can override the VoxelSize with your own value.

So it works like we agreed, with an added possibility to add your own number, which is practically not used in our code. I can of course create an extra parameter for the estimated voxel-size if you want, but I found this way more transparent and simple to read.

Reply:
OK. can do